dir.props

```diff
@@ -67,6 +67,9 @@
     <PackagesBinDir>$(__PackagesBinDir)</PackagesBinDir>
     <PackagesBinDir Condition="'$(__PackagesBinDir)'==''">$(BinDir).nuget\</PackagesBinDir>
 
+    <PackageInstallPath>$(__PackageInstallPath)</PackageInstallPath>
```

Comment:
OK. I guess you guys don't have a packages.props file like corefx. I would just put it where ever your other packaging properties are getting set.